docs/manual/java/gettingstarted/GettingStartedSbt.md

```diff
@@ -12,7 +12,7 @@ Follow these steps to create your first Lagom build:
 
 ## Create a new Lagom build
 
-From the command line, invoke `sbt new` and specify the Lagom Giter8 template:
+Open a terminal or command window with administrative privileges. From the command line, invoke `sbt new` and specify the Lagom Giter8 template:
```

Comment:
"... administrative privileges" <-- really? To create a new project user must be admin? I think this is wrong: either the docs or the fact that we require admin privileges (which I think we don't).